app/search.py

```diff
@@ -197,7 +197,8 @@ def _swiss_search(self):  # pylint: disable=too-many-branches, too-many-statemen
             coords = self._get_geoanchor_from_bbox()
             self.sphinx.SetGeoAnchor('lat', 'lon', coords[1], coords[0])  # pylint: disable=unsubscriptable-object
             self.sphinx.SetSortMode(sphinxapi.SPH_SORT_EXTENDED, '@geodist ASC')
-            limit = self.BBOX_SEARCH_LIMIT
+            limit = self.limit if self.limit and \
```

Comment:
cool, thank you :+1:

Reply:
could be simplified by `limit = min(self.limit, self.BBOX_SEARCH_LIMT)` Of course for this to work both parameter needs to be numbers